utils/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+pub const PUBLIC_PARAMS_BENCH_PATH: &str = "/var/tmp/lurk_benches/public_params";
+// Note: Must be called before `LURK_CACHE_CONFIG` is initialized,
+// e.g. by calling `public_params()`, in order to override
+// the preconfigured or default location
+pub fn set_public_params_dir(path: &str) {
```

Comment:
This looks out of place too. Can we put it somewhere that's more directly related to the configuration of public parameters?